### PR TITLE
inspectrum 0.2 (new formula)

### PR DESCRIPTION
**Pull request by bkonosky**
inspectrum: New formula for inspectrum, an offline radio signal analyzer.

- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/inspectrum.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (DomT4) [inline Formula/inspectrum.rb]: Don't need this empty line.

File: Formula/inspectrum.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  homepage "https://github.com/miek/inspectrum"
+  url "https://github.com/miek/inspectrum/archive/v0.2.tar.gz"
+  sha256 "50b7db9b86208f414c387700a358eb58364094f3e8a4985f586f4f815645898a"
+  head "http://github.com/miek/inspectrum.git"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "qt"
+  depends_on "fftw"
+  depends_on "liquid-dsp"
```
Comment (fxcoudert) [inline Formula/inspectrum.rb]: Please reorder (alphabetically) these 3 runtime deps

File: Formula/inspectrum.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system bin/"inspectrum", "-h"
```
Comment (fxcoudert) [inline Formula/inspectrum.rb]: Can we actually get it to do something on the command-line?

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/inspectrum.rb (hunk 3 above)
Comment (bkonosky) [reply Formula/inspectrum.rb]: No, running the program just opens a GUI, except for the -h switch, which just prints the help and exits.

Comment (fxcoudert) [reply Formula/inspectrum.rb]: Then can we at least match some output from `inspectrum -h`, like a version number?

File: Formula/inspectrum.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  depends_on "liquid-dsp"
+  depends_on "qt"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    assert_equal "Usage: /usr/local/Cellar/inspectrum/0.2/bin/inspectrum [options] file\nspectrum viewer\n\nOptions:\n  -h, --help       Displays this help.\n  -r, --rate <Hz>  Set sample rate.\n\nArguments:\n  file             File to view.", shell_output("#{bin}/inspectrum -h").strip
```
Comment (fxcoudert) [inline Formula/inspectrum.rb]: `  * C: 20: col 191: Line is too long. [286/190]`

Also, the test will not work if `HOMEBREW_PREFIX` is different that `/usr/local`.
Maybe we can just match a substring, like `--rate <Hz>  Set sample rate`

**Review by fxcoudert — APPROVED** [review]

File: Formula/inspectrum.rb [hunk 5]
```diff
@@ -0,0 +1,22 @@
+class Inspectrum < Formula
+  desc "Offline radio signal analyser"
+  homepage "https://github.com/miek/inspectrum"
+  url "https://github.com/miek/inspectrum/archive/v0.2.tar.gz"
+  sha256 "50b7db9b86208f414c387700a358eb58364094f3e8a4985f586f4f815645898a"
+  head "http://github.com/miek/inspectrum.git"
```
Comment (DomT4) [inline Formula/inspectrum.rb]: `https://`

Comment (fxcoudert) [reply Formula/inspectrum.rb]: I'll update that when I merge (although it's weird audit didn't flag it)

Comment (DomT4) [reply Formula/inspectrum.rb]: We also don't audit to check that GitHub `head` URLs end in `.git` so. There are gaps that could be filled, for sure.

**Comment by fxcoudert** [comment]
Congratulations @bkonosky on your first contribution to Homebrew!

**Pull request closed without merging**